AtReconstruction/AtFitter/OpenKF/kalman_filter/TrackFitterUKFTest.cxx

```diff
@@ -0,0 +1,268 @@
+#include "kalman_filter/TrackFitterUKF.h"
+
+#include "gtest/gtest.h"
+
+class TrackFitterUKFExampleTest : public testing::Test {
```

Comment:
This tests the code with a simple 2D model (constant acceleration). It's here to make sure we won't break anything really (and I didn't break anything when I was adapting the OpenKF code. This test is from them)